### ISSUE TITLE
QFSFileEngine::open: No file name specified

### ISSUE THREAD

**Issue by yurivict** [jamulussoftware/jamulus#1311]
**Describe the bug**
3.7.0 prints these error messages during regular activity like connecting to Jam rooms:
```
QFSFileEngine::open: No file name specified
QFSFileEngine::open: No file name specified
```

OS: FreeBSD 12.2

**Comment by jujudusud** [comment]
Same thing apprends on Archlinux.

**Comment by gilgongo** [comment]
This has been the case for most other releases. I believe it's not a problem but agree it would be nice not to have.

**Comment by dcorson-ticino-com** [comment]
I think this has something to do with setting stylesheets in Qt.
I can make it worse, but I have not been able to reduce their number.
Note that there are also messages coming from some Qt type library that seem to be caused by some fonts being used by people in their user names.

**Comment by gilgongo** [comment]
I think unless this is leading to some negative consequence, we can close this.

**Comment by softins** [comment]
This was fixed in #1514
